### ISSUE TITLE
Relicense AsyncSSH to EPL 2.0

### ISSUE THREAD

**Issue by ronf** [ronf/asyncssh#162]
I've received a request (#161) to consider relicensing AsyncSSH from the Eclipse Public License (EPL) version 1.0 to EPL 2.0, with an explicit provision for GPL 2.0 to be allowed as a secondary license. I would like to proceed with this, and ask all other contributors (@alexander-travov @jwilk @minesworld @parmentelat @viotti @wallies @ZhukovAlexander) to indicate your approval of this process by leaving a quick approval note here, or any questions or concerns you have about this.

More information about this process can be found at https://www.eclipse.org/legal/epl-2.0/faq.php#h.cktcxkzh8ks4.

The proposed new license header will look something like:

```python
# Copyright (c) 2013-2018 by Ron Frederick <ronf@timeheart.net> and others.
#
# This program and the accompanying materials are made available under
# the terms of the Eclipse Public License v2.0 which accompanies this
# distribution and is available at:
#
#     http://www.eclipse.org/legal/epl-2.0/
#
# This program may also be made available under the following secondary
# licenses when the conditions for such availability set forth in the
# Eclipse Public License v2.0 are satisfied:
#
#    GNU General Public License, Version 2.0, or any later versions of that license
#
# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only
```

**Comment by alexander-travov** [comment]
I approve.

**Comment by jwilk** [comment]
I don't think I contributed anything copyrightable, but for avoidance of doubt, I hereby agree to any relicensing.

>```
>#    GNU General Public License, Version 2.0, or any later versions of that license
>#
># SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only
>```

The first line says “2.0, or any later versions”, but the other one says “2.0-only”.
So which one is it?

**Comment by parmentelat** [comment]
I don't think either that my micro-contribution is worth me having any word in this discussion; 
please go ahead whichever way you see fit

**Comment by ZhukovAlexander** [comment]
I approve.

**Comment by wallies** [comment]
I approve

**Comment by ronf** [comment]
Good catch, @jwilk, and thanks all for the quick response!

The header text should be:

```python
# Copyright (c) 2013-2018 by Ron Frederick <ronf@timeheart.net> and others.
#
# This program and the accompanying materials are made available under
# the terms of the Eclipse Public License v2.0 which accompanies this
# distribution and is available at:
#
#     http://www.eclipse.org/legal/epl-2.0/
#
# This program may also be made available under the following secondary
# licenses when the conditions for such availability set forth in the
# Eclipse Public License v2.0 are satisfied:
#
#    GNU General Public License, Version 2.0, or any later versions of that license
#
# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-or-later
```

The exact copyright year range will vary depending on the module. I'll get this change in soon.

**Comment by ronf** [comment]
This change is now in the "develop" branch via commit 0b6eb7a. I'll be moving it to master and cutting the 1.14.0 release with this in the next few days.

**Comment by ronf** [comment]
These changes are now available in AsyncSSH 1.14.0.
